### PR TITLE
Calendar button

### PR DESCRIPTION
**Pull request by brianteeman**
Makes the calendar button accessible - this is used in the publishing fields and the custom date field

Adds aria-hidden=true
adds a title attribute to the button

### CODE REVIEW

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on db6314c7737a6440e74962a70e4dc4dda5124813<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19118">issues.joomla.org/tracker/joomla-cms/19118</a>.</sub>

File: administrator/language/en-GB/en-GB.lib_joomla.ini [hunk 1]
```diff
@@ -386,6 +386,7 @@ JLIB_HTML_BEHAVIOR_HOLD_MOUSE="- Hold mouse button on any of the buttons above f
 JLIB_HTML_BEHAVIOR_MONTH_SELECT="- Use the < and > buttons to select month\n"
 JLIB_HTML_BEHAVIOR_NEXT_MONTH_HOLD_FOR_MENU="Select to move to the next month. Select and hold for a list of the months."
 JLIB_HTML_BEHAVIOR_NEXT_YEAR_HOLD_FOR_MENU="Select to move to the next year. Select and hold for a list of years."
+JLIB_HTML_BEHAVIOUR_OPEN_CALENDAR="Open the calendar"
```
Comment (Quy) [inline administrator/language/en-GB/en-GB.lib_joomla.ini]: Change spelling `BEHAVIOUR`

File: layouts/joomla/form/field/calendar.php [hunk 2]
```diff
@@ -127,7 +127,8 @@
 			data-only-months-nav="<?php echo $singleheader; ?>"
 			<?php echo !empty($minYear) ? 'data-min-year="' . $minYear . '"' : ''; ?>
 			<?php echo !empty($maxYear) ? 'data-max-year="' . $maxYear . '"' : ''; ?>
-		><span class="icon-calendar"></span></button>
+			title="<?php echo JText::_('JLIB_HTML_BEHAVIOUR_OPEN_CALENDAR'); ?>"
+		><span class="icon-calendar" aria-hidden="true"></span></button>
```
Comment (Quy) [inline layouts/joomla/form/field/calendar.php]: Does it need the text also as an invisible element?

Comment (brianteeman) [reply layouts/joomla/form/field/calendar.php]: No because there is a title element

File: administrator/language/en-GB/en-GB.lib_joomla.ini (hunk 1 above)
Comment (brianteeman) [reply administrator/language/en-GB/en-GB.lib_joomla.ini]: Grrh I hate american

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 01c079423e80896856bbf0aa2f612b227fde49cb<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19118">issues.joomla.org/tracker/joomla-cms/19118</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 01c079423e80896856bbf0aa2f612b227fde49cb<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19118">issues.joomla.org/tracker/joomla-cms/19118</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**